lib/quest/rspec_runner.rb

```diff
@@ -0,0 +1,35 @@
+module Quest
+  class RSpecRunner
+    # Based loosely on https://github.com/guard/guard-rspec/blob/master/lib/guard/rspec/rspec_process.rb
+
+    attr_reader :result
+
+    def initialize(spec_file, spec_helper, tmp_status_file)
+      @spec_file       = spec_file
+      @spec_helper     = spec_helper
+      @tmp_status_file = tmp_status_file
```

Comment:
Personally, I would make this object utterly independent of others. Instead of working with some path that was passed in, I would have *this* object build its own tempfile, do its thing, then dispose of its own tempfile.

Essentially, something like:

``` Ruby
Tempfile.open('quest-rspec-runner') do |output|
    @exit_code  = run_spec(spec_file, spec_helper, output)
    @result     = read_result(output)
end


```